installer/pkg/config-generator/generator.go

```diff
@@ -19,6 +20,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/openshift/installer/installer/pkg/config"
+	"github.com/openshift/installer/pkg/rhcos"
```

Comment:
I think you need `github.com/openshift/installer/pkg/rhcos/ami` here (I'm leaving space for other RHCOS lookups besides Amazon's AMIs).

Reply:
I modified your change so that it was an `AMI` method in a new `rhcos` package. That way it reads `rhcos.AMI()` instead of `ami.AMI()` without aliasing the package. I can change that back if you want though.

Reply:
> I modified your change so that it was an AMI method in a new `rhcos` package. That way it reads `rhcos.AMI()` instead of `ami.AMI()` without aliasing the package.

Ah, that's fine with me.  My only concern would be if one particular RHCOS-lookup required more dependencies (e.g. if we needed a bunch of deps to find an AMI that we didn't need to find a QEMU image).  But we can always split it out into sub-packages if/when that happens ;).